File: BirthdayCakeCandles/src/main/java/MyCode.java
package main.java;

import java.util.Collections;
import java.util.List;

public class MyCode {

    public static int birthdayCakeCandles(List<Integer> candles) {
        Collections.sort(candles);
        int theHighest = candles.get(candles.size()-1);
        return (int) candles.stream().filter(it-> it == theHighest).count();
    }
}

File: BirthdayCakeCandles/src/main/kotlin/MyCode.kt
package main.kotlin

fun birthdayCakeCandles(candles: Array<Int>): Int {
    candles.sort()
    return candles.count { it -> it == candles.last() }

}

fun main(args: Array<String>) {
    val candlesCount = readLine()!!.trim().toInt()

    val candles = readLine()!!.trimEnd().split(" ").map{ it.toInt() }.toTypedArray()

    val result = birthdayCakeCandles(candles)

    println(result)
}